README.md

```diff
@@ -21,8 +21,9 @@ import (
 // When we use `wmi.CreateQuery` the name of the struct should match querying
 // WMI class name.
 type Win32_Process struct {
-	ProcessId uint32
+	PID       uint32 `wmi:"ProcessId"`
```

Comment:
what happens if you duplicate tag on some field? 
`
PID                 uint32 `wmi:"ProcessId"`
ProcessId      uint32 `wmi:"ProcessId"`
`

Reply:
Both will be resolved to the same value.

We'll probably need some pre-validation of the structure as an enhancement, but don't sure we need it right now.